src/win/winpty.cc

```diff
@@ -106,9 +118,10 @@ static NAN_METHOD(PtyGetProcessList) {
   int actualCount = winpty_get_console_process_list(pc, processList, processCount, nullptr);
 
   v8::Local<v8::Array> result = Nan::New<v8::Array>(actualCount);
-  for (uint32_t i = 0; i < actualCount; i++) {
+  for (int i = 0; i < actualCount; i++) {
```

Comment:
```suggestion
  for (size_t i = 0; i < actualCount; i++) {
```

Reply:
`actualCount` is an `int` because the library returns it as one. In that case, I'd rather keep `i` as an `int` than cast `actualCount` to `size_t` in case `actualCount` is negative for some reason.